### DISCUSSION TITLE
About the price

### DISCUSSION THREAD

**Discussion by juanbaqui** [opengisch/QField#4042] [Q&A]
I have a question about the price when creating an organization, for example, if I create an organization and in the project within the organization I add 10 people who are going to go out into the field and work in qfield, in this case I have to pay 16 euros for each participant? 160 euros per month in total?

**Comment by lucienicolier** [comment]
Hi @juanbaqui,

Thanks for your question.
You are right, if all your users are active during the month, you will pay for 10 users.
If, for example, you have only 5 members who are active during one month, you will only pay for those 5 even if you have 10 members in your organization.

Is this answer to your question?
